### PR TITLE
Moving to 10.2.0 update-1

### PR DESCRIPTION
**Pull request by Tanvi-Bhandari**
Changes include:
1) Applied Security patch on all AMIs - (Spectre Meltdown Vulnerability
Update)
2) New AMI for all 15 regions for INFAEICADMINHVM and  INFAEICCLUSTERHVM

3) Adding Confirm password parameter for DBPassword and
InformaticaAdminPassword.
4) Added Constraints for certain input parameters.
5) Added default value for IP Address range.
6) Added support for Cloud-watch group - For this template is going to
refer configuration file hosted by Informatica in public s3 bucket at
location :
https://s3.amazonaws.com/eic-cloudwatch-configuration/awsCloudwatchLogs.conf

AWS QS team can recommend on what is the best way to put the
configuration file for cloud-watch.

### CODE REVIEW

File: templates/informatica-eic-master.template [hunk 1]
```diff
@@ -232,6 +267,7 @@
         "IPAddressRange": {
             "Description": "The CIDR IP range that is permitted to access the Informatica domain and the Informatica embedded cluster",
             "Type": "String",
+            "Default": "0.0.0.0/0",
```
Comment (vsnyc) [inline templates/informatica-eic-master.template]: Could you remove this? The users must provide an input for this

File: templates/informatica-eic.template [hunk 2]
```diff
@@ -2413,13 +2539,13 @@
                                                 "Ref": "InformaticaAdminPassword"
                                             },
                                             "\n",
-                                            "echo /opt/informatica/isp/bin/infacmd.sh as createService -dn Domain -nn Node -sn Analyst_Service -un $administratorName -pd $administratorPassword -rs Model_Repository_Service -ds Data_Integration_Service -ffl /tmp -cs Catalog_Service -csau $administratorName -csap $administratorPassword -au $administratorName -ap $administratorPassword -bgefd /tmp -HttpPort 8089 >> /installation.log \n",
+                                            "echo \" /opt/informatica/isp/bin/infacmd.sh as createService -dn Domain -nn Node -sn Analyst_Service -un $administratorName -pd ****** -rs Model_Repository_Service -ds Data_Integration_Service -ffl /tmp -cs Catalog_Service -csau $administratorName -csap ****** -au $administratorName -ap ****** -bgefd /tmp -HttpPort 8089 \" >> /installation.log \n",
```
Comment (vsnyc) [inline templates/informatica-eic.template]: Are the arg values intentional? `-pd ******`, `-csap ******`, `-ap ******`
Previously it was: `-pd $administratorPassword`, `-csap $administratorPassword`, `-ap $administratorPassword`

File: templates/informatica-eic-master.template [hunk 3]
```diff
@@ -151,19 +169,34 @@
...
-            "Type": "String"
+            "Type": "String",
+            "Default": "Administrator",
+            "MinLength": "1",
+            "ConstraintDescription": "Informatica Admin Username is mandatory"
         },
         "InformaticaEICKeyName": {
             "Type": "String",
-            "Description": "The Informatica Enterprise Information Catalog license key name. For example, <license key name> or <bucket sub folder/license key name>"
+            "Description": "The Informatica Enterprise Information Catalog license key name. For example, <license key name> or <bucket sub folder/license key name>",
```
Comment (vsnyc) [inline templates/informatica-eic-master.template]: Should the description now state "Enterprise Data Catalog" and be corrected throughout the two templates?

File: templates/informatica-eic-master.template [hunk 4]
```diff
@@ -127,15 +135,25 @@
         },
         "DBPassword": {
             "Description": "Password for the database instance associated with Informatica domain and services (such as Model Repository Service, Data Integration Service, Content Management Service)",
```
Comment (vsnyc) [inline templates/informatica-eic-master.template]: update the description to match the constraint: "Password should be at least 8 characters long and cannot contain `<fill characters here>`"

File: templates/informatica-eic-master.template [hunk 5]
```diff
@@ -127,15 +135,25 @@
...
-            "MaxLength": "18",
+            "MaxLength": "128",
             "MinLength": "8",
             "NoEcho": "True",
+            "AllowedPattern": "[^/\"@]*",
+            "ConstraintDescription": "Database Password should be atleast 8 characters and cannot contain \" @ and ^",
```
Comment (vsnyc) [inline templates/informatica-eic-master.template]: Can you double check the constraint description? Not allowed characters are: `/`, `"`, `@`

File: templates/informatica-eic.template (hunk 2 above)
Comment (Tanvi-Bhandari) [reply templates/informatica-eic.template]: Not that it is echo statement to be redirected in /installation.log. Earlier the echo statement was printing all passwords in /installation.log. Now we have masked the sensitive info which is getting printed in /installation.log.

**Pull request closed without merging**